cloudfunction/pushToElasticSearch.js

```diff
@@ -0,0 +1,92 @@
+/**
+* Triggered from a message on a Cloud Pub/Sub topic.
+*
+* @param {!Object} event Event payload.
+* @param {!Object} context Metadata for the event.
+*/
+
+const mysql = require('promise-mysql');
+const axios = require('axios');
+const SHOP_SELECT_STATEMENT = `SELECT MerchantID, ShopID, ShopName, Latitude, Longitude, AddressLine1, TypeOfService FROM Shops WHERE ShopID=?;`;
+const MERCHANT_SELECT_STATEMENT = `SELECT MerchantName, MerchantPhone FROM Merchants WHERE MerchantID=?;`
+const CATALOG_SELECT_STATEMENT = `SELECT ServiceName from Catalog WHERE ShopID = ?;`
+const GCE_URL = 'http://10.128.0.13:9200'
```

Comment:
Can you comment or make the variable name more elaborate for this. GCE_URL is pretty generic and doesn't tell anything

Reply:
Okay. Will do that. Will ELASTICSEARCH_SERVER_URL be better?